be/src/exec/workgroup/work_group.cpp

```diff
@@ -351,11 +372,12 @@ void WorkGroupManager::apply(const std::vector<TWorkGroupOp>& ops) {
     // collect removable workgroups
     while (it != _workgroup_expired_versions.end()) {
         auto wg_it = _workgroups.find(*it);
-        DCHECK(wg_it != _workgroups.end());
-        if (wg_it->second->is_removable()) {
+        if (wg_it != _workgroups.end() && wg_it->second->is_removable()) {
+            int128_t wg_id = *it;
             _sum_cpu_limit -= wg_it->second->cpu_limit();
             _workgroups.erase(wg_it);
             _workgroup_expired_versions.erase(it++);
+            LOG(INFO) << "cleanup expired workgroup version:  " << (int64_t)(wg_id >> 64) << "," << (int64_t)wg_id;
```

Comment:
Will print a lot of logs?

Reply:
Only log at workgroup DDL (create/alter/delete).